src/components/InactivePastMonthLearnersTable/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+import TableContainer from '../../containers/TableContainer';
+import { formatTimestamp, formatPercentage } from '../../utils';
+import EnterpriseDataApiService from '../../data/services/EnterpriseDataApiService';
+
+const InactivePastMonthLearnersTable = () => {
+  const tableColumns = [
+    {
+      label: 'Email',
+      key: 'user_email',
+      columnSortable: true,
+    },
+    {
+      label: 'Course Title',
+      key: 'course_title',
+      columnSortable: true,
+    },
+    {
+      label: 'Course Price',
+      key: 'course_price',
+      columnSortable: true,
+    },
+    {
+      label: 'Start Date',
+      key: 'course_start',
+      columnSortable: true,
+    },
+    {
+      label: 'End Date',
+      key: 'course_end',
+      columnSortable: true,
+    },
+    {
+      label: 'Passed Date',
```

Comment:
I believe the inactive tables (both past week & past month) should not have the `Passed Date` table column per the "No has passed column" in the ticket.